src/quo/theme.cljs

```diff
@@ -59,3 +59,42 @@
   [component]
   (fn [& args]
     (into [:f> f-with-theme component] args)))
+
+(def previous-screen (atom nil))
+
+(def ^:private change-statusbar-color-delay 150)
+
+(def ^:private dark-screens
+  #{:shell :share-shell :activity-center :camera-screen :settings :settings-password :settings-syncing
+    :edit-profile})
+
+(def ^:private ignored-screens
+  #{:chats-stack :communities-stack :wallet-stack :browser-stack :shell})
+
+(defn handle-status-bar-screens-per-theme
+  []
+  (js/setTimeout #(rn/set-status-bar-style
+                   (if (dark?)
+                     "light-content"
+                     "dark-content")
+                   true)
+                 change-statusbar-color-delay))
+
+(defn handle-status-bar-dark-screen
+  []
+  (js/setTimeout #(rn/set-status-bar-style "light-content" true) change-statusbar-color-delay))
+
```

Comment:
tbh I dont think these files really relate to "theme".
I think you should make a new file for these and put them in somewhere like navigation/status_bar.cljs 👍 
I think it would be better suited as it's very specific to the application where as quo/theme is a generic theme utility 👍